### ISSUE TITLE
install node-avro-io@0.3.3 is trying to install outdated int64-native

### ISSUE THREAD

**Issue by jcollum** [jamesbrucepower/node-avro-io#28]
```
$ npm install node-avro-io@0.3.3 --save

> int64-native@0.3.2 install /Users/collumj/work/projects-akmak/crdb-avro-converter/node_modules/node-avro-io/node_modules/int64-native
> node-gyp configure && node-gyp build

gyp WARN download NVM_NODEJS_ORG_MIRROR is deprecated and will be removed in node-gyp v4, please use NODEJS_ORG_MIRROR
gyp WARN download NVM_NODEJS_ORG_MIRROR is deprecated and will be removed in node-gyp v4, please use NODEJS_ORG_MIRROR
gyp WARN download NVM_NODEJS_ORG_MIRROR is deprecated and will be removed in node-gyp v4, please use NODEJS_ORG_MIRROR
  CXX(target) Release/obj.target/Int64/src/main.o
In file included from ../src/main.cc:8:
../src/Int64.h:10:22: error: expected class name
class Int64 : public ObjectWrap {
                     ^
../src/Int64.h:22:34: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> New(const Arguments& args);
                                 ^~~~~
```

etc. 

I'm not sure why; the package.json is asking for the right version. I can install the int64-native@0.4.x just fine on OSX but the 0.3 version fails.

**Comment by jcollum** [comment]
I pulled the code down and `npm pack`ed it and it works fine. Is it possible the version on npm is incorrect?

**Comment by jcollum** [comment]
Confirmed. The version on npm has 

```
 "dependencies": {
    "lodash": "^2.4.1",
    "snappy": "^2.1.1",
    "buffer-crc32": "^0.2.5",
    "int64-native": "0.3.2"
  },
```

int64-native@0.3.2 won't install on OS X El Cap / Node 4.4.4 / NPM 3.

**Comment by shellscape** [comment]
same problem here

**Comment by jcollum** [comment]
@shellscape  I found that https://github.com/mtth/avsc was a better fit for my project. Last commit 16 days ago.

**Comment by shellscape** [comment]
@jcollum thanks. tried that one too. unfortunately it fails very hard on complex schemas - opened an issue there.

**Comment by jcollum** [comment]
Did you try pulling the code down for this repo and packing and installing from the local file? I'd be surprised if this library was better at handling complex schemas; it appears to be a bit forgotten.

**Comment by shellscape** [comment]
Nah, I've just been trying out various avro libs. `etp-avro` has been the only one so far able of handling our schema.

**Comment by mdlavin** [comment]
This was fixed as part of PR #23
